pkg/appmanager/appManager_test.go

```diff
@@ -44,280 +45,56 @@ func init() {
+var oneIappOneNodeConfig string = readConfigFile(configPath + "/oneIappOneNodeConfig_exp.json")
 
-var twoSvcTwoPodsConfig string = string(`{"resources":{"velcro":{"virtualServers":[{"name":"default_barmap","pool":"/velcro/cfgmap_default_barmap_bar","ipProtocol":"tcp","enabled":true,"sourceAddressTranslation":{"type":"automap"},"destination":"/velcro/10.128.10.240:6051","profiles":[{"partition":"Common","name":"http","context":"all"},{"partition":"Common","name":"tcp","context":"all"}]},{"name":"default_foomap","pool":"/velcro/cfgmap_default_foomap_foo","ipProtocol":"tcp","enabled":true,"sourceAddressTranslation":{"type":"none"},"destination":"/velcro/10.128.10.240:5051","profiles":[{"partition":"Common","name":"http","context":"all"},{"partition":"Common","name":"tcp","context":"all"}]}],"pools":[{"name":"cfgmap_default_barmap_bar","loadBalancingMode":"round-robin","members":[{"address":"10.2.96.0","port":80,"session":"user-enabled"},{"address":"10.2.96.3","port":80,"session":"user-enabled"}]},{"name":"cfgmap_default_foomap_foo","loadBalancingMode":"round-robin","members":[{"address":"10.2.96.1","port":8080,"session":"user-enabled"},{"address":"10.2.96.2","port":8080,"session":"user-enabled"}]}]}}}`)
+var twoSvcTwoPodsConfig string = readConfigFile(configPath + "/twoSvcTwoPodsConfig_exp.json")
 
-var oneSvcTwoPodsConfig string = string(`{"resources":{"velcro":{"virtualServers":[{"name":"default_barmap","pool":"/velcro/cfgmap_default_barmap_bar","ipProtocol":"tcp","enabled":true,"sourceAddressTranslation":{"type":"automap"},"destination":"/velcro/10.128.10.240:6051","profiles":[{"partition":"Common","name":"http","context":"all"},{"partition":"Common","name":"tcp","context":"all"}]}],"pools":[{"name":"cfgmap_default_barmap_bar","loadBalancingMode":"round-robin","members":[{"address":"10.2.96.0","port":80,"session":"user-enabled"},{"address":"10.2.96.3","port":80,"session":"user-enabled"}]}]}}}`)
+var oneSvcTwoPodsConfig string = readConfigFile(configPath + "/oneSvcTwoPodsConfig_exp.json")
+
+func readConfigFile(path string) string {
+	data, err := ioutil.ReadFile(path)
+	// Returns emptyConfig is the file cannot be read
+	if nil != err {
+		return string(`{"resources":{}}`)
```

Comment:
Instead of returning an empty config here leverage ginkgo and use Expect(err).To(BeNil()) so that way we can catch the test failed since it tried to load a non-existent file rather than have the test failed because we ran with an empty config.